clang/lib/Driver/Driver.cpp

```diff
@@ -3486,8 +3486,17 @@ class OffloadingActionBuilder final {
           }
           continue;
         }
+        ActionList DeviceLibObjects;
+        ActionList LinkObjects;
+        for (const auto &I : LI) {
```

Comment:
Minor readability suggestion:
```suggestion
        for (const auto &Input : LI) {
```

Reply:
Done.